README.md

```diff
@@ -47,7 +47,38 @@ The following features can be toggled in [application.conf](conf/application.con
 
 ### Navigation
 
-Navigation is handled using the default HMRC style with a Navigator class defining normal and check routes
+Navigation is handled using multiple modes to facilitate additions, amendments, and removals at the various data levels.
+
+The two modes are `NormalMode` and `CheckMode`. `CheckMode` is indicative of something being *changed*, and can be seen by the presence of `change` in the URL.
+
+Examples:
+1. Changing an additional reference in a consignment item:
+   
+   I am *changing* a house consignment by *changing* a consignment item by *changing* an additional reference
+   
+   So my URL would follow the strcuture of: `.../change-house-consignment/:houseConsignmentIndex/change-item/:itemIndex/change-additional-reference/:additionalReferenceIndex/...`
```

Comment:
Tiny typo on structure which occurs a few times in the README

Reply:
Good spot, thank you